bentoml/_internal/frameworks/fastai.py

```diff
@@ -0,0 +1,285 @@
+    from ..models.model import ModelSignaturesType
+
+try:
+    import torch
+    import torch.nn as nn
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
+        "fastai requires `torch` as a dependency. Please follow PyTorch instruction at https://pytorch.org/ in order to use `fastai`."
+    )
+try:
+    from fastai import __version__ as FASTAI_VERSION  # type: ignore
+    from fastai.learner import Learner
+    from fastai.learner import load_learner  # type: ignore
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
+        "fastai is required in order to use module `bentoml.fastai`, install fastai with `pip install fastai`. For more information, refers to https://docs.fast.ai/#Installing."
+    )
+
+try:
+    import fastai.basics  # type: ignore
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
+        "BentoML will only support fastai v2 onwards. Please uninstall fastai v1 and try again. Please visit https://docs.fast.ai/#Installing for more information."
+    )
+
+
+__all__ = ["load_model", "save_model", "get_runnable", "get"]
+
+
+def get(tag_like: str | Tag) -> bentoml.Model:
+    """
+    Get the BentoML model with the given tag.
+
+    Args:
+        tag_like (``str`` ``|`` :obj:`~bentoml.Tag`):
+            The tag of the model to retrieve from the model store.
+    Returns:
+        :obj:`~bentoml.Model`: A BentoML :obj:`~bentoml.Model` with the matching tag.
+
+    Example:
+
+    .. code-block:: python
+
+        import bentoml
+        # target model must be from the BentoML model store
+        model = bentoml.fastai.get("fai_learner")
+    """
+    model = bentoml.models.get(tag_like)
+    if model.info.module not in (MODULE_NAME, __name__):
+        raise NotFound(
+            f"Model {model.tag} was saved with module {model.info.module}, failed loading with {MODULE_NAME}."
+        )
+    return model
+
+
+def load_model(
+    bento_model: str | Tag | bentoml.Model,
+    cpu: bool = True,
+    pickle_module: types.ModuleType = cloudpickle,
+) -> Learner:
+    """
+    Load the :code:`fastai.learner.Learner` model instance with the given tag from the local BentoML model store.
+
+    Args:
+        bento_model (``str`` ``|`` :obj:`~bentoml.Tag` ``|`` :obj:`~bentoml.Model`):
+            Either the tag of the model to get from the store, or a BentoML `~bentoml.Model`
+            instance to load the model from.
+        cpu (``bool``, `optional`, default to ``True``):
+            Whether to load the model to CPU. if ``True``, the ``nn.Module`` is mapped to ``cpu`` via :code:`map_location` in ``torch.load``.
+            The loadded dataloader will also be converted to ``cpu``.
+
+            .. admonitions:: About :code:`cpu=True`
+
+                If model uses **mixed_precision**, then the results models will also be converted to FP32. Learn more about `mixed precision <https://docs.fast.ai/callback.fp16.html>`.
```

Comment:
mixed_precision looks like code; if it's not maybe just say "if the model uses **mixed precision**"?

```suggestion
                If the model uses ``mixed_precision``, then the loaded model will also be converted to FP32. Learn more about `mixed precision <https://docs.fast.ai/callback.fp16.html>`.
```

Reply:
I want this to be bold instead of double backtick.